tests/target/closure_block_style.rs

```diff
@@ -0,0 +1,25 @@
+// rustfmt-indent_style: Block
+// rustfmt-version: Two
+
+fn foo() {
+    {
+        let write_status = |
+            status: &mut Vec<ansi_term::ANSIString>,
+            diff: &Diff,
+            heading: &str,
+            color: &Style,
+            show_hints: bool,
+            hints: &[&str]
```

Comment:
We should add a trailing comma here by default. If the `trailing_comma` is set to `Never`, then we should not add a trailing comma.

Reply:
Done!